src/main/java/io/neonbee/endpoint/odatav4/internal/olingo/processor/CountEntityCollectionProcessor.java

```diff
@@ -102,7 +97,8 @@ public void readEntityCollection(ODataRequest request, ODataResponse response, U
             ContentType responseFormat) {
         List<UriResource> resourceParts = uriInfo.getUriResourceParts();
         if (resourceParts.size() > 2) {
-            throw TOO_MANY_PARTS_EXCEPTION;
+            throw new UnsupportedOperationException(
+                    "Read requests with more than two resource parts are not supported.");
```

Comment:
What is the benefit of removing the constant?

Reply:
According to Error Prone it is a code smell, see [1]

[1] https://errorprone.info/bugpattern/StaticAssignmentOfThrowable

Reply:
Ah cool, I wasn't aware of this. Of course if this produces always the same stack trace it is bad.